www/js/survey/input-matcher.js

```diff
@@ -2,23 +2,37 @@
 
 import angular from 'angular';
 
-angular.module('emission.survey.inputmatcher', ['emission.plugin.logger'])
-.factory('InputMatcher', function(Logger){
-  var im = {};
-
-  const EPOCH_MAXIMUM = 2**31 - 1;
-  const fmtTs = function(ts_in_secs, tz) {
-    return moment(ts_in_secs * 1000).tz(tz).format();
-  }
-
-  var printUserInput = function(ui) {
-    return fmtTs(ui.data.start_ts, ui.metadata.time_zone) + "("+ui.data.start_ts + ") -> "+
-           fmtTs(ui.data.end_ts, ui.metadata.time_zone) + "("+ui.data.end_ts + ")"+
-           " " + ui.data.label + " logged at "+ ui.metadata.write_ts;
-  }
-
-  im.validUserInputForDraftTrip = function(trip, userInput, logsEnabled) {
-    if (logsEnabled) {
+angular
+  .module('emission.survey.inputmatcher', ['emission.plugin.logger'])
+  .factory('InputMatcher', function (Logger) {
+    var im = {};
+
+    const EPOCH_MAXIMUM = 2 ** 31 - 1;
+    const fmtTs = function (ts_in_secs, tz) {
+      return moment(ts_in_secs * 1000)
+        .tz(tz)
+        .format();
+    };
+
+    var printUserInput = function (ui) {
+      return (
+        fmtTs(ui.data.start_ts, ui.metadata.time_zone) +
```

Comment:
It's kind of odd how Prettier formats all of these multi-line print statements.
Is there anything we can do to make it less aggressive with changing line breaks?